pallets/nft/src/lib.rs

```diff
@@ -1470,4 +1470,19 @@ impl<T: Config> NFTTrait<T::AccountId, BalanceOf<T>> for Pallet<T> {
 
 		Ok(minted_token_id)
 	}
+
+	fn get_stackable_nft_balance(who: &T::AccountId, asset_id: &(Self::ClassId, Self::TokenId)) -> BalanceOf<T> {
+		NftModule::<T>::get_stackable_collections_balances((asset_id.0, asset_id.1, who))
+	}
+
+	fn transfer_stackable_nft(
```

Comment:
As Reserved stackable nft has already implemented, we need to prevent it's being transferred as well.

Reply:
should be resolved in the following commit

Reply:
fixed in the latest commit